frontend/src/components/browse-sightings-page/SightingCard.tsx

```diff
@@ -29,11 +40,15 @@ export const SightingCard: React.FunctionComponent<SightingProps> = ({
 
       <div className="card-data fade-in">
         <p>
-          <span className="sighting-card-information">Description: </span>
+          <span className="sighting-card-information">
+            <strong>Description of sighting: </strong>
```

Comment:
Don't use `<strong>` for styling purposes - it's a semantic element with a very specific use-case! If you want bold text, style the `<span>`:
```suggestion
            Description of sighting:
```

Reply:
sorted, thanks!